docs/Components.md

```diff
@@ -0,0 +1,67 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Components
+
+## Components of Basic Calico Container Deployments
+
+All basic Calico deployments require the following components:
+
+ - **`calico/node`**: Docker image that runs the underlying calico processes (see 
+   ***Anatomy of calico/node*** below). The `calico/node` image must be deployed 
+   on each host in the network.
+
+ - **`calicoctl`**: The command line tool responsible for configuring all of the 
+   networking components within Calico, including endpoint addresses, security 
+   policies, and BGP peering.
+
+ - **Orchestrator (Docker, Kubernetes, etc)**: Manages container creation/deletion 
+   and runs the `calico/node` Docker image as a container.
+
+ - [**etcd**](https://github.com/coreos/etcd): Datastore used by Calico to store 
+   endpoint, bgp, and policy data that can be accessed by all of the hosts.  
+   See the [etcd Data Model page](./etcdStructure.md) for details on how Calico 
+   data is stored in etcd. 
+
+## Anatomy of calico/node
+
+`calico/node` can be regarded as a helper container that bundles together the 
+various components required for Calico networking.  The image utilizes the 
+following processes:
+
+#### Calico Felix agent
+
+The Felix daemon is the heart of Calico networking.  Felix's primary job is to 
+program routes and ACL's on a workload host to provide desired connectivity to 
+and from workloads on the host.  Felix programs 
+endpoint routes of workloads into the host's Linux kernel FIB table so that packets 
+to endpoints arrive on the host and then forward to the correct endpoint.
+
+Felix also programs interface information to the kernel for outgoing endpoint 
+traffic. Felix instructs the host to respond to ARPs for workloads with the 
+MAC address of the host.
+
+#### BIRD internet routing daemon
+
+BIRD is an open source BGP client that is used to exchange routing information 
+between hosts.  The routes that Felix programs into the kernel for endpoints 
+are picked up by BIRD and distributed to BGP peers on the network, which 
+provides inter-host routing.
+
+#### confd templating engine 
+
+The confd templating engine watches the etcd datastore for any changes to BGP 
+configuration.  Confd dynamically generates BIRD configuration files based on 
```

Comment:
The [confd README](https://github.com/kelseyhightower/confd/blob/master/README.md) only uses lowercase 'c'; we should do the same.